src/async_impl/request.rs

```diff
@@ -101,6 +103,18 @@ impl Request {
         &mut self.body
     }
 
+    /// Get the response body limit.
+    #[inline]
+    pub fn response_body_limit(&self) -> Option<&usize> {
```

Comment:
I kinda think because it's just a number, people would expect more that it just returns a copy, so `Option<u64>`.